### PR TITLE
[DOCFIX] Improve UFS ozone doc

### PR DESCRIPTION
**Pull request by xichen01**
### What changes are proposed in this pull request?
Improve UFS ozone doc, Added:
- the mount about HA Ozone mode 
- the mount of the specified Ozone version

### Why are the changes needed?

Improve UFS ozone doc

### Does this PR introduce any user facing changes?
NO

### CODE REVIEW

**Review by maobaolong — CHANGES_REQUESTED** [review]
Left couple of comments, please take a look, otherwise, it LGTM.

File: docs/cn/ufs/Ozone.md [hunk 1]
```diff
@@ -29,15 +29,48 @@ Ozone可以在容器化环境(例如Kubernetes和YARN)中有效运行。
...
+alluxio.master.mount.table.root.ufs=o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>.<OM_SERVICE_IDS>/
+alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration=/path/to/hdfs/conf/ozone-site.xml
 ``` 
 
+`<OM_SERVICE_IDS>` 可以在`ozone-site.xml`中找到
+可能的 `ozone-site.xml` 文件, 从文件中可以知道, `<OM_SERVICE_IDS>` 为 `tdw`
+```xml
+<property>
+    <name>ozone.om.service.ids</name>
+    <value>tdw</value>
```
[the remark this answers is not in the document]
Comment (xichen01) [reply docs/cn/ufs/Ozone.md]: done

File: docs/en/ufs/Ozone.md [hunk 2]
```diff
@@ -30,15 +30,48 @@ To configure Alluxio to use Ozone as under storage, you will need to modify the
...
+alluxio.master.mount.table.root.ufs=o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>.<OM_SERVICE_IDS>/
+alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration=/path/to/hdfs/conf/ozone-site.xml
 ``` 
 
+`<OM_SERVICE_IDS>` can be found in `ozone-site.xml`
+Possible `ozone-site.xml` file, as known from the file, `<OM_SERVICE_IDS>` is `tdw`
+```xml
+<property>
+    <name>ozone.om.service.ids</name>
+    <value>tdw</value>
```
[the remark this answers is not in the document]
Comment (xichen01) [reply docs/en/ufs/Ozone.md]: done

**Review by maobaolong — APPROVED** [review]
LGTM. Thanks @xichen01

**Comment by maobaolong** [comment]
@jiacheliu3 Would you like to have another look?

File: docs/cn/ufs/Ozone.md [hunk 3]
```diff
@@ -25,19 +25,53 @@ Ozone可以在容器化环境(例如Kubernetes和YARN)中有效运行。
...
 ```
 
-编辑`conf/alluxio-site.properties`文件把底层存储地址设置为Ozone桶和 
-想要挂载到Alluxio的Ozone目录。例如，如果要将整个存储桶挂载到Alluxio
-底层存储的地址可以是`o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/`
-，或者是`o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/alluxio/data`如果仅将`<OZONE_VOLUME>`的 `<OZONE_BUCKET>` ozone桶内的`/alluxio/data`目录映射到Alluxio。
+编辑`conf/alluxio-site.properties`将`alluxio.master.mount.table.root.ufs`底层存储地址设置为Ozone桶或想要挂载到Alluxio的Ozone目录。
+例如，如果要将整个存储桶挂载到Alluxio底层存储的地址可以是`o3fs://<OZONE_BUCKET>.<s_VOLUME>/`
+如果仅将`<OZONE_VOLUME>`的`<OZONE_BUCKET>`ozone桶内的`/alluxio/data`目录映射到Alluxio。such
+底层存储的地址可以是`o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/alluxio/data`。
```
Comment (dbw9580) [inline docs/cn/ufs/Ozone.md:35]:
```suggestion
如果仅将`<OZONE_VOLUME>`的`<OZONE_BUCKET>`ozone桶内的`/alluxio/data`目录映射到Alluxio的根目录，
则可以设置为`o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/alluxio/data`。
```

File: docs/cn/ufs/Ozone.md [hunk 4]
```diff
@@ -25,19 +25,53 @@ Ozone可以在容器化环境(例如Kubernetes和YARN)中有效运行。
...
+要让Alluxio挂载HA模式下Ozone，你应该配置Alluxio的服务端以让其可以找到OzoneManager。请注意一旦设置，你使用Alluxio客户端的应用程序不再需要任何特殊的配置。
+在HA模式下`alluxio.master.mount.table.root.ufs`需要指定`<OM_SERVICE_IDS>`
+例如：
+```properties
+alluxio.master.mount.table.root.ufs=o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>.<OM_SERVICE_IDS>/
+alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration=/path/to/hdfs/conf/ozone-site.xml
+``` 
+
+`<OM_SERVICE_IDS>` 可以在`ozone-site.xml`中找到，
+可能的 `ozone-site.xml` 文件，从该文件中可以知道，`<OM_SERVICE_IDS>` 为 `omservice1`。
```
Comment (dbw9580) [inline docs/cn/ufs/Ozone.md:55]:
```suggestion
例如以下`ozone-site.xml`配置文件中`<OM_SERVICE_IDS>` 为 `omservice1`。
```

File: docs/cn/ufs/Ozone.md [hunk 5]
```diff
@@ -79,6 +113,14 @@ $ ./bin/alluxio fs mount \
   /ozone o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/
 ```
 
+如果需要挂载指定版本的Ozone可以通过`alluxio.underfs.version=<OZONE_VERSION>`指定。
```
Comment (dbw9580) [inline docs/cn/ufs/Ozone.md:116]:
```suggestion
如果需要挂载指定版本的Ozone，可以在挂载时通过`alluxio.underfs.version=<OZONE_VERSION>`参数指定版本。
```

File: docs/en/ufs/Ozone.md [hunk 6]
```diff
@@ -26,19 +26,52 @@ to install a Ozone cluster, and follow the [Cli Commands](https://ozone.apache.o
...
+To make Alluxio mount Ozone in HA mode, you should configure Alluxio's server so that it can find the OzoneManager. Please note that once set up, your application using the Alluxio client does not require any special configuration.
+In HA mode `alluxio.master.mount.table.root.ufs` needs to specify `<OM_SERVICE_IDS>`
+such as:
+```properties
+alluxio.master.mount.table.root.ufs=o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>.<OM_SERVICE_IDS>/
+alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration=/path/to/hdfs/conf/ozone-site.xml
+``` 
+
+`<OM_SERVICE_IDS>` can be found in `ozone-site.xml`
+Possible `ozone-site.xml` file, as known from the file, `<OM_SERVICE_IDS>` is `omservice1`
```
Comment (dbw9580) [inline docs/en/ufs/Ozone.md:55]:
```suggestion
`<OM_SERVICE_IDS>` can be found in `ozone-site.xml`.
In the following example `ozone-site.xml` file, `<OM_SERVICE_IDS>` is `omservice1`:
```

File: docs/en/ufs/Ozone.md [hunk 7]
```diff
@@ -26,19 +26,52 @@ to install a Ozone cluster, and follow the [Cli Commands](https://ozone.apache.o
...
+```xml
+<property>
+    <name>ozone.om.service.ids</name>
+    <value>omservice1</value>
+</property>
+```
+
+### Mount HDFS with Specific Versions
+
+User can mount an Ozone cluster with a specified version as an under storage into Alluxio namespace.
```
Comment (dbw9580) [inline docs/en/ufs/Ozone.md:65]:
```suggestion
Users can mount an Ozone cluster of a specific version as an under storage into Alluxio namespace.
```

File: docs/en/ufs/Ozone.md [hunk 8]
```diff
@@ -83,6 +116,14 @@ $ ./bin/alluxio fs mount \
   /ozone o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/
 ```
 
+If you need mount an HDFS cluster with a specified version, you can specify it through `alluxio.underfs.version=<OZONE_VERSION>`.
```
Comment (dbw9580) [inline docs/en/ufs/Ozone.md:119]:
```suggestion
If you need to mount an Ozone cluster of a specific version, you can specify it through the command line option `alluxio.underfs.version=<OZONE_VERSION>`.
```

**Review by dbw9580 — CHANGES_REQUESTED** [review]

File: docs/cn/ufs/Ozone.md [hunk 9]
```diff
@@ -25,19 +25,53 @@ Ozone可以在容器化环境(例如Kubernetes和YARN)中有效运行。
...
+```console
 $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 ```
 
-编辑`conf/alluxio-site.properties`文件把底层存储地址设置为Ozone桶和 
-想要挂载到Alluxio的Ozone目录。例如，如果要将整个存储桶挂载到Alluxio
-底层存储的地址可以是`o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/`
-，或者是`o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/alluxio/data`如果仅将`<OZONE_VOLUME>`的 `<OZONE_BUCKET>` ozone桶内的`/alluxio/data`目录映射到Alluxio。
+编辑`conf/alluxio-site.properties`将`alluxio.master.mount.table.root.ufs`底层存储地址设置为Ozone桶或想要挂载到Alluxio的Ozone目录。
+例如，如果要将整个存储桶挂载到Alluxio底层存储的地址可以是`o3fs://<OZONE_BUCKET>.<s_VOLUME>/`
```
[the remark this answers is not in the document]
Comment (xichen01) [reply docs/cn/ufs/Ozone.md]: done

File: docs/cn/ufs/Ozone.md (hunk 3 above)
Comment (xichen01) [reply docs/cn/ufs/Ozone.md]: done

File: docs/cn/ufs/Ozone.md (hunk 4 above)
Comment (xichen01) [reply docs/cn/ufs/Ozone.md]: done

File: docs/cn/ufs/Ozone.md (hunk 5 above)
Comment (xichen01) [reply docs/cn/ufs/Ozone.md]: done

File: docs/en/ufs/Ozone.md (hunk 6 above)
Comment (xichen01) [reply docs/en/ufs/Ozone.md]: done

File: docs/en/ufs/Ozone.md (hunk 7 above)
Comment (xichen01) [reply docs/en/ufs/Ozone.md]: done

File: docs/en/ufs/Ozone.md (hunk 8 above)
Comment (xichen01) [reply docs/en/ufs/Ozone.md]: done

**Review by dbw9580 — APPROVED** [review]
LGTM

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**